### PR TITLE
Spec refactor

### PR DESCRIPTION
**Pull request by dreeve**
@supertopher Any chance you could take a look at this?

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@dreeve Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=pivotal-cloudops/secrets-sanitizer&pullRequestId=12)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=pivotal-cloudops/secrets-sanitizer&pullRequestId=12) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/faq) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@dreeve Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=pivotal-cloudops/secrets-sanitizer&pullRequestId=12)!

**Comment by pivotal-issuemaster** [comment]
@dreeve Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=pivotal-cloudops/secrets-sanitizer&pullRequestId=12)!

File: spec/desanitize_executor_spec.rb [hunk 1]
```diff
@@ -41,17 +40,24 @@
   end
 
   after do
-   FileUtils.rm_r tmp_dir
+    FileUtils.rm_r tmp_dir
```
Comment (supertopher) [inline spec/desanitize_executor_spec.rb]: crazy town right here 👀

**Comment by supertopher** [comment]
👍

**Pull request merged**